dist/build.mjs

```diff
@@ -60,6 +60,31 @@ class Build {
     }
   }
 
+  static checkArgs () {
+    console.log('check args')
+    if (process.argv.length <= 5) {
```

Comment:
shouldn't this test be for < 5 rather than <= 5?

Reply:
Yes, will fix it